modules/battery_manager/help/battery_manager.md

```diff
@@ -0,0 +1,63 @@
+# Battery Manager
+
+The Battery Manager module serves as a front-end for browsing, adding, editing, and activating/deactivating entries in the Test Battery.
+
+## Searching for an entry in the Test Battery
+
+Under the `Browse` tab, use the `Selection Filters` to search for entries by fields such 
+as `Instrument`, `Minimum age (days)`, `Maximum age (days)`, `Stage`, `Subproject`, `Visit Label`,
+`Site`, `First Visit`, `Instrument Order`, and `Active`. As filters are selected, the data table below will
+dynamically update with relevant results. Click the “Clear Filters”
```

Comment:
suggest to put `Clear Filters` in bold instead than between quotes to be as other references to button on this page.